components/AppBar/src/MDCAppBar.m

```diff
@@ -18,13 +18,13 @@
 
 #import "MDCAppBarContainerViewController.h"
 
+#import "MDFTextAccessibility.h"
 #import "MaterialFlexibleHeader.h"
 #import "MaterialIcons+ic_arrow_back.h"
 #import "MaterialRTL.h"
 #import "MaterialShadowElevations.h"
 #import "MaterialShadowLayer.h"
 #import "MaterialTypography.h"
-#import "MDFTextAccessibility.h"
```

Comment:
What did you use to format these? At Google capital letters sort before lowercase...